packages/dappmanager/src/daemons/telegram/telegramCommands.ts

```diff
@@ -0,0 +1,127 @@
+import TelegramBot from "node-telegram-bot-api";
+import * as db from "../../db";
+import { logs } from "../../logs";
+import { bold, buildTelegramMessage, url } from "./buildTelegramMessage";
+import { commandsList } from "./commandsList";
+
+/**
+ * Polls for commands and responds.
+ * Channel ID is not needed to response to messages
+ * @param telegramToken
+ */
+export async function telegramCommands(bot: TelegramBot): Promise<void> {
+  try {
+    bot.startPolling();
+    // POLLING ERRORS
+    // 1. EFATAL if error was fatal e.g. network error
+    // 2. EPARSE if response body could not be parsed
+    // 3. ETELEGRAM if error was returned from Telegram servers
+    // ETELEGRAM: 409 Conflict  =>  More than one bot instance polling
+    // ETELEGRAM: 404 Not Found => wrong token or not found
+    bot.on("polling_error", error => {
+      throw Error(`${error.name}: ${error.message}`);
+    });
+
+    // Listen for any messages. If channel ID does not exists, it saves the channel ID
+    bot.on("message", async msg => {
+      if (msg.text && commandsList.includes(msg.text) === false)
+        try {
+          const chatId = msg.chat.id.toString();
+          const channelExists = checkIfChannelIdExists(chatId);
+          if (channelExists === false) {
+            const message = buildTelegramMessage({
+              telegramMessage: "Succesfully saved channel ID",
+              telegramMessageType: "Success"
+            });
+            subscribeChannelId(chatId);
+            await sendTelegramMessage({ bot, chatId, message });
+          }
+        } catch (e) {
+          logs.error("Error on telegram daemon. message polling", e);
+        }
+    });
+
+    // Remove channel ID
+    bot.onText(/\/unsubscribe/, async msg => {
+      try {
+        const chatId = msg.chat.id.toString();
+        let message = "";
+        const channelExists = checkIfChannelIdExists(chatId);
+        if (channelExists === true) {
+          unsubscribeChannelId(chatId);
+          message = buildTelegramMessage({
```

Comment:
For bot messages don't use the same format as a notification. They should appear clearly different

Reply:
As the previous conversation, I believe that the formatHeader() creates different headers for different types of messages. There are notifications , help, alert, etc.